kernel/buffer.c

```diff
@@ -0,0 +1,70 @@
+#include "declarations.h"
+#include "functions.h"
+
+// Initialize Buffer Cache
```

Comment:
You can add a small description of what this file is responsible for

Reply:
I'll do it